package.json

```diff
@@ -112,6 +113,7 @@
     "typeface-roboto": "0.0.54",
     "typescript": "^3.2.2",
     "uglify-es": "^3.3.9",
+    "uuid": "^3.3.2",
```

Comment:
Since this is part of the component, this will need to go in `dependencies` so it is bundled with the package.

Reply:
Done